packages/blockprotocol/cli/print-spacer.js

```diff
@@ -0,0 +1,9 @@
+import chalk from "chalk";
+
+export const printSpacer = () => {
+  console.log(
+    chalk.bold(
+      "––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––",
```

Comment:
I guess we can use `process.stdout.rows` here as a cap (if defined).

Reply:
https://github.com/blockprotocol/blockprotocol/pull/575/commits/d7c8886fa34d7792f4861a587fe2e137ecb69aa1